### PR TITLE
Move podlogs into e2e/storage/testsuites

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

The e2e framework package podlogs is used in e2e/storage/testsuites
only. In addition we considered we should have a single e2e framework
package for pod without the podlogs. So this moves the podlogs into
e2e/storage/testsuites.

This PR comes from discussion of https://github.com/kubernetes/kubernetes/pull/86679

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
/lgtm
/priority backlog
cc @pohly

**Comment by oomichi** [comment]
@neolit123
Thanks for your review again, I did mistake on golint before.

**Comment by neolit123** [comment]
@oomichi yep, saw the package name change.

File: test/e2e/storage/testsuites/podlogs.go [hunk 1]
```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
...
+// Package testsuites enables live capturing of all events and log
 // messages for some or all pods in a namespace as they get generated.
 // This helps debugging both a running test (what is currently going
 // on?) and the output of a CI run (events appear in chronological
 // order and output that normally isn't available like the command
 // stdout messages are available).
-package podlogs
+package testsuites
 
 import (
```
Comment (pohly) [inline test/e2e/storage/testsuites/podlogs.go]: The functions in `podlogs.go` have nothing to do with the functions in the `testsuites` package. If the code really isn't welcome in `test/e2e/framework` anymore, then at least keep in a separate package. I suggest `test/e2e/storage/podlogs`.

**Review by pohly — CHANGES_REQUESTED** [review]

File: test/e2e/storage/testsuites/podlogs.go (hunk 1 above)
Comment (oomichi) [reply test/e2e/storage/testsuites/podlogs.go]: @pohly Thanks for your suggestion.
OK, I will make it as `test/e2e/storage/podlogs` today.

/hold

Comment (oomichi) [reply test/e2e/storage/testsuites/podlogs.go]: done

/hold cancel

**Review by alejandrox1 — COMMENTED** [review]
If these functions aren't used anywhere else then guess this is a good step to cleaning up the e2e framework.
And seems like they landed in their own pkg within e2e/storage.

/lgtm

**Pull request merged**